.github/workflows/ci.yaml

```diff
@@ -0,0 +1,33 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - 'master'
+
+jobs:
+  clone:
+    runs-on: ubuntu-latest
```

Comment:
im not sure i understand how this works 
is this setup on this repo because i didnt see it in the list of runners

Reply:
The migration to minutes was okay. This keyword is defined in https://github.com/actions/virtual-environments#available-environments and uses the minutes one. It uses ubuntu-20.04 and the setup commands will run on that instance.

You can view the minutes balance at https://github.com/organizations/bonlineza/settings/billing
It's an ubuntu 2 core machine. We have 3000 minutes, and have so far used 6.

You can also view the available tags and active jobs at https://github.com/organizations/bonlineza/settings/actions/hosted-runners

Reply:
I see 
I was being dumb 
i understand